src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -77,6 +79,18 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 	});
 };
 
+export const generateBranchName = (issueKey: string, issueSummary: string) => {
+	if (!issueSummary) {
+		return `${issueKey}-`;
+	}
+
+	let branchSuffix = issueSummary;
```

Comment:
I know this block is a bit unsophisticated lol, but its intention is clear and I didnt want to refactor things or us ,.apply etc not today